src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -5,19 +5,19 @@
 public enum KeyBindingCategory {
 
     FILE(
-            Localization.menuTitle("File")),
+            Localization.lang("File")),
```

Comment:
Yes it is but the translation strings have a leading and ("&") sign and since most of the strings are also in the non menu localization files I changed it.

On Thu, Sep 08, 2016 at 10:30 PM, Oliver Kopp < notifications@github.com [notifications@github.com] > wrote:
In src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java [https://github.com/JabRef/jabref/pull/1940#discussion_r78082976] :

> @@ -5,19 +5,19 @@
>  public enum KeyBindingCategory {
>  
>      FILE(
> -            Localization.menuTitle("File")),
> +            Localization.lang("File")),


Why this change? Isn't the category mirroring the menu structure? Menu strings are translated in the menu translation files, not the normal language files.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/JabRef/jabref/pull/1940/files/e5277fac194f6ec75df9d693b4c5845511a679a6#r78082976] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AOoU1Zr8Cm_Ea1R-87FnLK3v3kFPVkaXks5qoHBegaJpZM4J4D-T] .